src/vs/platform/audioCues/browser/audioCueService.ts

```diff
@@ -43,13 +45,17 @@ export class AudioCueService extends Disposable implements IAudioCueService {
 	constructor(
 		@IConfigurationService private readonly configurationService: IConfigurationService,
 		@IAccessibilityService private readonly accessibilityService: IAccessibilityService,
-		@ITelemetryService private readonly telemetryService: ITelemetryService,
-	) {
+		@ITelemetryService private readonly telemetryService: ITelemetryService) {
```

Comment:
Nit (not important to fix): the trailing comma and uniform formatting in the original code before this change makes diffs and editing (using multi-cursor) easier :)